src/modules/auth/auth.service.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  BadRequestException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { LoginDto } from './dto/login.dto';
+import * as bcryptjs from 'bcryptjs';
+import { JwtService } from '@nestjs/jwt';
+import { ResestPasswordDto } from './dto/resetPassword.dto';
+import { UsersService } from '../users/users.service';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private readonly jwtService: JwtService,
+    private readonly usersSevice: UsersService,
+  ) {}
+
+  async login(data: LoginDto): Promise<{ access_token: string }> {
+    const user = await this.usersSevice.findOneByEmail(data.email);
+    if (!user) {
+      throw new UnauthorizedException('Email does not exist');
```

Comment:
in the constant file, can you create a String file? and put this string error message there, then we can use it across the app the same string